handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,72 @@
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](../../../company/remote/index.md) that is [distributed across the world](../../../company/team/index.md). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](../../people-ops/compensation.md) and [practical benefits](../../people-ops/benefits-and-perks.md) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](../../index.md)
+
+## Interview process
+
+1. You apply here (TODO - create Lever job)
+1. Intro calls
+   - 45m [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) (VP Engineering)
+   - 30m [Beyang Liu](../../../company/team/index.md#beyang-liu) (CTO)
+1. Interviews.
+   - 1h [Team collaboration](https://github.com/sourcegraph/interviews/blob/master/engineering/team-collaboration.md). We ask you about how you work and communicate in a team setting, and how you handle tricky situations.
+     - The interviewers will be [Christina Forney](../../../company/team/index.md##christina-forney-she-her) (VP Product) and one of the following:
+       - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
+       - [Eric Broody-Moore](../../../company/team/index.md#eric-brody-moore)
+       - [María Craig](../../../company/team/index.md#maría-craig-she-her)
+       - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
+   - 1h [Engineering leadership](engineering-leadership.md).
+     - The interviewers will be [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) and [Tomás Senart](../../../company/team/index.md#tomás-senart).
+   - 1h [Technical experience](https://github.com/sourcegraph/interviews/blob/master/engineering/technical-experience.md) - We ask you about your past work and accomplishments.
+     - The interviewers will be two of the following:
+       - [Ryan Slade](../../../company/team/index.md#ryan-slade-he-him)
+       - [Alex Russell-Saw](../../../company/team/index.md#alex-russell-saw-he-him)
+       - [Thorsten Ball](../../../company/team/index.md#thorsten-ball-he-him)
+   - 30m [Quinn Slack](../../../company/team/index.md#quinn-slack) (CEO)
+
+If we want to move forward after these sets of interviews, we will setup 30-minute calls to meet any [members of the Cloud team](../cloud/index.md#members) that weren't already an interviewer. These calls are informal and don't have a script.
+
+We want to ensure Sourcegraph is an environment that suits your working style and empowers you to do your best work, so we are eager to answer any questions that you have about us at any point in the interview process.
+
+Apply here. — TODO Create Lever job
```

Comment:
```suggestion
<!-- **[Click here to apply](#todo)** --><!-- This is commented out because I will be sending some outbound messages before accepting applications. -->
```